django/cantusdb_project/static/js/melody_search.js

```diff
@@ -72,41 +72,42 @@ function melodySearch() {
 
     function trackClick(event) {
         const y = event.pageY;
+        y_rel = y - drawAreaTop;
         // add the note to the search term, according to the y position of the click
         if (index <= 14) {
-            if (y < 264) {
+            if (y_rel < 22) {
                 notes += "q";
```

Comment:
Maybe we can leave this to a future PR, but it could be good to put this mapping in an object ("dictionary") - it would eliminate a bunch of `else if`s, and ensure that the values are always identical between `trackClick` and `trackMouse`.

Reply:
Nvm...will make the change here.